### ISSUE TITLE
The default sort, Smart Sort, prioritizes user count far too much over active users and comments

### ISSUE THREAD

**Issue by zabadoh** [tgxn/lemmy-explorer#187]
Now that Lemmy is almost a year old, the user base has settled down and found their places, it's time to look at whether the default presentation can be made more useful for new users to find relevant and active communities.

I have found that the default sort, Smart Sort, weights user count far too highly over active users and comments.

When Lemmy had its first user migration waves, only a few communities with the right keywords were there, and the new users flocked to those /c's.

Whether it's from moderator neglect or abuse, or someone decided they could do better, some communities have migrated to other /c's or other instances where there are more active membership.

But these migrated active communities lack the membership counts of the older, neglected communities, and therefore appear far lower than the older communities that were the beneficiaries of the initial user waves.

**Additional context**
For example:  "movies"

A search for "movies" with the default Smart Sort yields the the following top 5 communities:

#1 !movies@lemmy.world - 6.9k subscribers, 813 posts, 8.6k comments, 1.4k Active Users (week)

#2 !star_wars@lemmy.world - 4.7k subscribers, 314 posts, 2.7k comments, 290 Active Users (week)

#3 !sciencefiction@lemmy.world - 13.2k subscribers, 305 posts, 7.9k comments, 68 Active Users (week)

#4 !bmoviebonanza@lemmy.world - 1.9k subscribers, 425 posts, 1.8k comments, 85 Active Users (week)

#5 !movies@lemm.ee - 1.2k subscribers, 931 posts, 6.9k comments, 1.9k Active Users (week)

While I love Star Wars, science fiction, and B movies, the paltry numbers of active users in communities #2, 3, and 4 compared to #1 and #5 indicate they really shouldn't be the #2, 3, and 4 recommendations for new users looking for places to discuss movies...

One can find patterns like that searching for many common interests...

Why is this important?

Because Lemmy Explorer is frequently recommended for new Lemmy users, and the default Smart Sort isn't giving the most relevant active communities up top.

**Comment by tgxn** [comment]
Yes, I need to find time to dive into refactoring the scoring methods




Currently, they use some ~sketchy~ complex algo that determines the score based on:

Instance Score: https://github.com/tgxn/lemmy-explorer/blob/main/crawler/src/output/trust.js#L531
- instances this instance is linked with / 2 (having a linked instance gives you a point for each link)
- instance endorsements * 4 (having endorsements will boost you in the search results)
- instances that allow federation for this instance * 3 (each allowed instance gives you points)
- **minus** instances that defederated with this instance * 10 (each blocked instance takes away points)
- 
Community Score: https://github.com/tgxn/lemmy-explorer/blob/main/crawler/src/output/trust.js#L584
```js
    const activityScore = community.counts.subscribers;
    const activeScore = community.counts.users_active_month * 20;
    const score = instanceMetrics.score * activityScore * activeScore;
```

**Comment by tgxn** [comment]
New Sort Method for Communities deployed here

https://develop.lemmyverse.net/communities?query=movies
